ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -52,8 +65,61 @@ private MigrationConfig(final Map<String, String> configs, final String id) {
             "",
             Importance.HIGH,
             "The URL for the KSQL server"
-        )
-        .define(
+        ).define(
+            KSQL_USERNAME,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The username for the KSQL server"
+        ).define(
+            KSQL_PASSWORD,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The password for the KSQL server"
+        ).define(
+            SSL_TRUSTSTORE_LOCATION,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The trust store path"
+        ).define(
+            SSL_TRUSTSTORE_PASSWORD,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The trust store password"
+        ).define(
+            SSL_KEYSTORE_LOCATION,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The key store path"
+        ).define(
+            SSL_KEYSTORE_PASSWORD,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The key store password"
+        ).define(
+            SSL_KEY_PASSWORD,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The key password"
+        ).define(
+            SSL_KEY_ALIAS,
+            Type.STRING,
+            null,
+            Importance.MEDIUM,
+            "The key alias"
+        ).define(
+            SSL_ALPN,
+            Type.STRING,
+            false,
+            Importance.MEDIUM,
+            "Whether ALPN should be used. It defaults to false."
+        ).define(
```

Comment:
Let's also add a config to control whether hostname verification is enabled or not.